### PR TITLE
React, Antd, Redux: Left sidebar and top for annotation page

### PR DESCRIPTION
**Pull request by bsekachev**
SVG icons now are a part of the bundle

### CODE REVIEW

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-side-bar.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                {/* eslint-disable-next-line */}
+                <span
+                    className={`cvat-annotation-page-objects-sidebar
+                        ant-layout-sider-zero-width-trigger
+                        ant-layout-sider-zero-width-trigger-left`}
+                    onClick={
+                        (): void => this.setState({ collapsed: !collapsed })
+                    }
+                >
+                    {collapsed && <Icon type='menu-fold' title='Show' />}
```
Comment (mmarkelov) [inline cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-side-bar.tsx]:
```javascript
{collapsed ? <Icon type='menu-fold' title='Show' /> : <Icon type='menu-unfold' title='Hide' />}
```

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-side-bar.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                trigger={null}
+                collapsed={collapsed}
+            >
+                {/* eslint-disable-next-line */}
+                <span
+                    className={`cvat-annotation-page-objects-sidebar
+                        ant-layout-sider-zero-width-trigger
+                        ant-layout-sider-zero-width-trigger-left`}
+                    onClick={
+                        (): void => this.setState({ collapsed: !collapsed })
```
Comment (mmarkelov) [inline cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-side-bar.tsx]: It's better to use function when you rely on [previous state](https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous):
```javascript
this.setState(prevState => ({ collapsed: !prevState.collapsed }))
```

File: cvat-ui/src/stylesheet.css [hunk 3]
```diff
@@ -101,53 +82,28 @@
 }
 
 
-.ant-menu.cvat-header-menu {
-    width: -webkit-fit-content;
-    width: -moz-fit-content;
-    width: fit-content;
+.ant-dropdown-trigger.cvat-header-menu-dropdown {
+    display: -webkit-box;
```
Comment (mmarkelov) [inline cvat-ui/src/stylesheet.css]: Don't you want to try [autoprefixer](https://github.com/postcss/autoprefixer) for adding prefixes?

Comment (bsekachev) [reply cvat-ui/src/stylesheet.css]: I am going to change css > less or sass soon, and use autoprefixer also

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-side-bar.tsx (hunk 2 above)
Comment (bsekachev) [reply cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-side-bar.tsx]: Interesting, thx

**Pull request merged**